models/attribute-definition/src/generator.ts

```diff
@@ -0,0 +1,28 @@
+import * as AttributeType from '@commercetools-test-data/attribute-type';
+import { LocalizedString } from '@commercetools-test-data/commons';
+import { fake, Generator } from '@commercetools-test-data/core';
+import { TAttributeDefinition } from './types';
+
+// https://docs.commercetools.com/api/projects/productTypes#attributedefinition
+
+const generator = Generator<TAttributeDefinition>({
+  fields: {
+    type: fake(() => AttributeType.random()),
+    name: fake((f) => f.commerce.productName()),
+    label: fake(() => LocalizedString.random()),
+    inputTip: fake(() => LocalizedString.random()),
+    isRequired: fake((f) => f.datatype.boolean()),
+    attributeConstraint: fake((f) =>
+      f.helpers.arrayElement([
```

Comment:
Constants can be used here too!

Reply:
fixed with [6c9a1d7](https://github.com/commercetools/test-data/pull/192/commits/6c9a1d7634cd6aa846c40e5ad73882d77ba4a585) 😎